daemon/daemon_unix.go

```diff
@@ -915,12 +915,13 @@ func setHostGatewayIP(controller *libnetwork.Controller, config *config.Config)
 func driverOptions(config *config.Config) nwconfig.Option {
 	return nwconfig.OptionDriverConfig("bridge", options.Generic{
 		netlabel.GenericData: options.Generic{
-			"EnableIPForwarding":  config.BridgeConfig.EnableIPForward,
-			"EnableIPTables":      config.BridgeConfig.EnableIPTables,
-			"EnableIP6Tables":     config.BridgeConfig.EnableIP6Tables,
-			"EnableUserlandProxy": config.BridgeConfig.EnableUserlandProxy,
-			"UserlandProxyPath":   config.BridgeConfig.UserlandProxyPath,
-			"Rootless":            config.Rootless,
+			"EnableIPForwarding":      config.BridgeConfig.EnableIPForward,
+			"EnableFilterForwardDrop": config.BridgeConfig.EnableFilterForwardDrop,
```

Comment:
I don't think we have docs for this in [man/dockerd.8.md](https://github.com/moby/moby/blob/6ac445c42badc98b416baa32cca5d6f8b712b359/man/dockerd.8.md) wondering if we should look at that in a follow-up 🤔 

Is this an option that can be "live reloaded" ? (i.e., enabled/disabled when reloading the daemon config?)

Reply:
Added to the manpage.

For config reload ... it's not, like `--ip-forward` and `--iptables`/`--ip6tables`. I think they're all similar in that they affect host config when a network's created, and that host config won't be un-done if the daemon config is changed.

Perhaps someone might want to fix the filter-FORWARD policy manually after creating a network, then reload daemon config so that it's not changed again when the next network is created. But, I think the same applies to IP forwarding and iptables and it's probably best that they all have the same behaviour?